src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -66,8 +66,7 @@ class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val we
   fun updateTier(tier: String, crn: String): ResponseEntity<Void>? {
     return updateTierCall(tier, crn)
       .also {
-        log.info("Updated Tier for $crn")
-        log.debug("Body: $tier for $crn")
+        log.info("Updated Tier for $crn with $tier")
```

Comment:
As below:

`I'm not sure we can log this at info because it has the CRN and the tier in the same message.
Anything logged at Info will end up in Application Insights, but Debug should only be output on dev machines`